packages/codemirror/src/language.ts

```diff
@@ -27,7 +27,7 @@ export class EditorLanguageRegistry implements IEditorLanguageRegistry {
     // Add default language text/plain -> No expressions to parse
     this.addLanguage({
       name: 'none',
-      mime: 'text/plain',
+      mime: IEditorMimeTypeService.defaultMimeType,
```

Comment:
I am not sure about this one. It could have unexpected consequences if someone changes `defaultMimeType` downstream.